client/app/dashboard/appSubmit.js

```diff
@@ -1,5 +1,78 @@
-angular.module('hunt.appSubmit', [])
+angular.module('hunt.appSubmit', ['hunt.backlog'])
 
-.controller('AppSubmitController', function ($scope) {
+.controller('AppSubmitController', function ($scope, AppSubmitFactory) {
+  $scope.appSubmitInfo = {};
+  $scope.appSubmitList = [];
+
+  $scope.getAppSubmits = function () {
+    AppSubmitFactory.findAll();
+  };
+
+  $scope.modify = function (target) {
+    AppSubmitFactory.edit(target);
+  };
+
+  $scope.removeApp = function (target) {
+    AppSubmitFactory.removeApp(target);
+  };
+
+
+  $scope.getAppSubmits();
+})
+
+.factory('AppSubmitFactory', function ($http) {
+
+  var findAll = function (callback) {
+    return $http({
+      method: 'GET',
+      url: '/api/appsubmits'
+    }).then(function (response) {
+      if (callback) {
+        return callback(response.data);
+      } else {
+        return response.data;
+      }
+    })
+  };
+
+  var addNew = function (data) {
+    return $http({
+      method: 'POST',
+      url: 'api/appsubmits',
+      data: data
+    }).then(function (response) {
+      response.status = 201
+      return response;
+    });
+  };
+
+  var removeApp = function (data) {
```

Comment:
Think we can remove this and combine functionality with edit. We'll still want separate actions in the controller though.